docs/how-to/default-profile.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Your profile defines which file-system and data-store your IPFS node will use, along with other configuration options. Learn how to set, change, and reset your profile.
+---
+
+# Configure a default profile and switching between profiles
+
+The profile of your IPFS node allows you to specify which file-system or data-store you want to use. Changing these options will affect the performance of your node. For example, if you want to have a faster datastore for your node, you can use the `badgerds` profile. If you're running your node on a low-power device like a Raspberry Pi, you can use the `low-power` profile. Profiles have been developed to customize your IPFS node to perform best under given conditions.
+
+## Find your current profile
+
+Your IPFS profile is found within your node's `config` file. The default location for the `config` file is `~/.ipfs/config`. If you have set an `$IPFS_PATH` variable, you can find your `config` file at `$IPFS_PATH/config`. Use `grep` to find the currently set profile:
+
+```bash
+cat ~/.ipfs/config | grep "prefix"
+# or
+cat $IPFS_PATH/config | grep "prefix"
+
+> "prefix": "flatfs.datastore",
+> "prefix": "leveldb.datastore",
+```
+
+IPFS uses the `flatfs` profile by default. `flatfs` is implemented using LevelDB, which is why you also see another line in the output with `leveldb.datastore`, but they both refer to `flatfs` datastore.
+
+If you previously configured your IPFS node to use another profile, let's say `badgerds`, the above command would output the following:
+
+```bash
+"prefix": "badger.datastore",
+```
+
+This output line refers to `badgerds` profile .
+
+## Available profiles
+
+Here's a list of all the profiles available for your IPFS node:
+
+- `flatfs`
+- `badgerds`
+- `server`
+- `randomports`
+- `default-datastore`
+- `local-discovery`
+- `test`
+- `default-networking`
+- `lowpower`
```

Comment:
@AlexxNica this was suggested by @johnnymatthews. Plus It's always better to avoid more user clicks. Redirecting the user to another page only to come back here seems to be a little over-do. @johnnymatthews would love your opinion on this.

Reply:
@jay-dee7, we may be better off by referencing to a single source of truth and having less technical debt related to maintenance. Besides, only having the available options doesn't tell the user what each option does and their reasoning.

But I can be wrong. What do you think?

Reply:
@jay-dee7, we may be better off by referencing to a single source of truth and having less technical debt related to maintenance. Besides, only having the available options doesn't tell the user what each option does and the reasoning behind them.

But I can be wrong. What do you think?

Reply:
@AlexxNica I totally believe that having a single source of truth definitely helps. Maybe we can migrate the entire doc from github readme to over here on [docs.ipfs.io](docs.ipfs.io). I'll be happy if the user finds everything under a single page tho

Reply:
You're both correct. Having all documentation in one place is the dream, but we don't want to increase our technical debt. A [project was created](https://github.com/ipfs/ipfs-docs/projects/3) towards the end of 2020 to start single-sourcing IPFS documentation. For now, I'm going to leave this list in, with the aim of having clearer explanations of these profiles in the future.